src/Executor/Values.php

```diff
@@ -94,7 +94,7 @@ public static function getVariableValues(Schema $schema, $definitionNodes, array
      * @return array
      * @throws Error
      */
-    public static function getArgumentValues($def, $node, $variableValues)
+    public static function getArgumentValues($def, $node, $variableValues = null)
```

Comment:
its a BC because the signature change, this class is not final then it can be override :)